### DISCUSSION TITLE
Blinking screen

### DISCUSSION THREAD

**Discussion by NeFender** [ppy/osu#19811] [Q&A] [answered]
### Type

Game behaviour

### Bug description

The screen starts blinking after unfolding the game. Sorry of bad English.

### Screenshots or videos
https://drive.google.com/file/d/1elFckN18tDi4Q6WEAnn38Woj_HgFQ0O2/view?usp=sharing

Sorry for bad quality. When recording the screen on the video, the blinking will not be visible.
### Version

2022.810.2

### Logs

[performance.log](https://github.com/ppy/osu/files/9351852/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9351853/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9351854/updater.log)
[database.log](https://github.com/ppy/osu/files/9351855/database.log)
[network.log](https://github.com/ppy/osu/files/9351856/network.log)

**Comment by ghost** [comment]
Sorry for the answer but the video link is liked to a file in your computer.

**Reply by NeFender** [reply]
I'll fix it now.

**Comment by smoogipoo** [comment, marked as answer]
This is likely going to be an issue with your graphics card/drivers. Please make sure you have all the latest drivers installed, and/or wait for us to switch to using DirectX for better compatibility with your system.
